lib/Dialect/Random/IR/RandomOps.td

```diff
@@ -0,0 +1,89 @@
+#ifndef LIB_DIALECT_RANDOM_IR_RANDOMOPS_TD_
+#define LIB_DIALECT_RANDOM_IR_RANDOMOPS_TD_
+
+include "lib/Dialect/Random/IR/RandomDialect.td"
+include "lib/Dialect/Random/IR/RandomTypes.td"
+include "mlir/IR/BuiltinAttributes.td"
+include "mlir/IR/CommonTypeConstraints.td"
+include "mlir/IR/OpBase.td"
+include "mlir/Interfaces/InferTypeOpInterface.td"
+include "mlir/Interfaces/SideEffectInterfaces.td"
+
+
+class Random_Op<string mnemonic, list<Trait> traits = [Pure]> :
+        Op<Random_Dialect, mnemonic, traits> {
+  let cppNamespace = "::mlir::heir::random";
+  let assemblyFormat = "operands attr-dict `:` functional-type(operands, results)";
+}
+
+def Random_InitOp : Random_Op<"init_prng"> {
+  let summary = "Initializes the PRNG with a seed";
+  let description = [{
+    Initializes the PRNG with a seed.
```

Comment:
could you add some description / justification for the PRNG here and/or in its type def? Something like that this instantiates a source of randomness that can be used to generate different types of random distributions. Also include maybe some suggestions on whether we expect this to be static and generated once globally (or not)? What kind of output range does it produce? (e.g. does it create uniformly distributed 64-bits, 32-bits)

The output range should probably be an attribute on the prng type, I think, and this op should probably take the number of bits as an attribute.